### PR TITLE
#162163132 Users should be able to Login via Google or Facebook 

### PR DESCRIPTION
**Pull request by alexkayabula**
## What does this PR do?
- The PR ensures that users can log in using their favourite social media platform.
### Description of task to be completed?
- Users should be able to Login via Google or Facebook.
### How should this be manually tested?
- Clone the repository.
- Install the dependencies using `yarn install`.
- Start the application using `yarn start`.
- A user should have a facebook or google account to sign in.
- Click the respective buttons to login via Facebook or Google.
screen shot 2019-01-22 at 10 37 41

 ## Login with Facebook.
screen shot 2019-01-18 at 07 50 10

## Login with Google.
screen shot 2019-01-18 at 07 48 08

### What are the relevant pivotal tracker stories?
[#162163132](https://www.pivotaltracker.com/story/show/162163132)

### CODE REVIEW

**Review by sekayasin — APPROVED** [review]
LGTM

**Review by barema4 — COMMENTED** [review]
Nice work, could you please add  `A user should have facebook or google account to signup` on how this task is manually tested

**Comment by alexkayabula** [comment]
> Nice work, could you please add `A user should have facebook or google account to signup` on how this task is manually tested

I think you meant "sign in". Let me work on that.

**Review by ekumamait — APPROVED** [review]
Nice work @alexkayabula looks good to merge.

**Review by ongebo — APPROVED** [review]

**Review by RrNn — CHANGES_REQUESTED** [review]
Move the `registerReducer.test.js` file to the `__tests__` directory

**Comment by alexkayabula** [comment]
> Move the `registerReducer.test.js` file to the `__tests__` directory

OK .... Let me do that

File: src/store/index.js [hunk 1]
```diff
@@ -11,4 +11,4 @@ const store = createStore(
   )
   
   );
-export default store;
+export default store;
```
Comment (RrNn) [inline src/store/index.js]: Add a new black line here

File: src/actions/actionTypes.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const actionTypes ={
+FACEBOOK_LOGIN: "FACEBOOK_LOGIN",
+GOOGLE_LOGIN: "GOOGLE_LOGIN",
+}
+
+export default actionTypes;
```
Comment (RrNn) [inline src/actions/actionTypes.js]: Add a new blank line here

**Review by RrNn — CHANGES_REQUESTED** [review]
Add new lines at the end of files.

File: src/store/index.js (hunk 1 above)
Comment (alexkayabula) [reply src/store/index.js]: I have made the changes.

File: src/actions/actionTypes.js (hunk 2 above)
Comment (alexkayabula) [reply src/actions/actionTypes.js]: I have made the changes

**Review by RrNn — APPROVED** [review]

**Pull request merged**